### DISCUSSION TITLE
is it possible to get details of number of mails delivered by specific IP of pool?

### DISCUSSION THREAD

**Discussion by jaydeepnp** [postalserver/postal#2582] [Help with using Postal]
Hi 

I have setup postal servers (2 Numbers) with IP Pool configuration (2 IPs on each pool). Total of 4 public IP Addresses are used to deliver emails.
 
Now I want get details on  counts of emails delivered using each IP Address.

Please help and confirm if this is possible, by seeing  databases tables, messages and few other not finding this information. 

If this is not available with current version is it possible to include this as enhancement request?

Regards,
Jaydeep

**Comment by willpower232** [comment]
I'm not sure if this information is available in the database but even if it is, Postal doesn't store information long term so would not be a good solution for tracking such information over time and you would need something else to read the Postal database(s) and store that information separately for as long as you want.
